packages/database/src/migrations/20210507014219-PutWaiviviaToTheRightPlace-modifies-data.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+import { codeToId, insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const hierarchyNameToId = async (db, name) => {
+  const record = await db.runSql(`SELECT id FROM entity_hierarchy WHERE name = '${name}';`);
+  return record?.rows[0].id;
+};
+
+const WAIVIVIA_CODE = 'FJ_WAV';
+const BURETA_CODE = 'FJ_CATCH_Bureta';
+const EASTERN_CODE = 'FJ_Eastern';
+const projectCode = 'wish';
+
+exports.up = async function (db) {
+  const WAIVIVIA_ID = await codeToId(db, 'entity', WAIVIVIA_CODE);
+  const BURETA_ID = await codeToId(db, 'entity', BURETA_CODE);
+
+  await db.runSql(`
+    UPDATE entity 
+    SET parent_id = '${BURETA_ID}'
+    WHERE code = '${WAIVIVIA_CODE}';
+  `);
+
+  await insertObject(db, 'entity_relation', {
```

Comment:
Any tiny bit of pickiness, if hierarchyNameToId doesn't find the names for whatever reason then this will insert a row with nulls or turn an error. Not a big deal if you know the entries are there but just thought Id point it out.

Reply:
Yeah I agree, will add an IF statement for safer inserting.